src/containers/landing/index.tsx

```diff
@@ -18,20 +18,22 @@ import { C4CState } from '../../store';
 interface LandingProps {
   readonly neighborhoods: MapGeoDataReducerState['neighborhoodGeoData'];
   readonly blocks: MapGeoDataReducerState['blockGeoData'];
+  readonly sites: MapGeoDataReducerState['siteGeoData'];
 }
 
 const mapStateToProps = (state: C4CState): LandingProps => {
   return {
     neighborhoods: state.mapGeoDataState.neighborhoodGeoData,
     blocks: state.mapGeoDataState.blockGeoData,
+    sites: state.mapGeoDataState.siteGeoData,
   };
 };
 
 const PaddedContent = styled.div`
   padding: 24px 50px;
 `;
 
-const Landing: React.FC<LandingProps> = ({ neighborhoods, blocks }) => {
+const Landing: React.FC<LandingProps> = ({ neighborhoods, blocks, sites }) => {
```

Comment:
❓ For all of these request props, this all looks good, but I only see code for the `asyncRequestIsComplete` state. What happens when any of these requests fail or are loading? Are we displaying that information to the user with a error message alert or spinner?

Reply:
No, as is the map view just wouldn't load. I'll open a separate ticket to handle these cases.